src/openms/include/OpenMS/KERNEL/BaseFeature.h

```diff
@@ -178,6 +192,34 @@ namespace OpenMS
     /// state of peptide identifications attached to this feature. If one ID has multiple hits, the output depends on the top-hit only
     AnnotationState getAnnotationState() const;
 
+    /// has a primary ID (peptide, RNA, compound) been assigned?
```

Comment:
I think if we keep both PeptideIdentifications and the new IDDatastructure we need to properly group them - like that I think nobody will figure out what belongs together